kamon-scala-future/src/main/scala/kamon/scala/FutureSpans.scala

```diff
@@ -0,0 +1,42 @@
+package kamon.scala
+
+import kamon.Kamon
+import kamon.Kamon.withContextKey
+import kamon.trace.Span
+import kamon.util.CallingThreadExecutionContext
+
+import scala.concurrent.Future
+
+object FutureSpans {
+  def withNewAsyncSpan[T](operationName: String, spanKind: String, component: String, tags: Map[String, String] = Map.empty)(code: Span ⇒ Future[T]): Future[T] = {
+    val spanBuilder = Kamon.buildSpan(operationName)
+      .withMetricTag("span.kind", spanKind)
+      .withTag("component", component)
```

Comment:
`component` should be metric tags as well. Please take a look at https://github.com/kamon-io/Kamon/issues/525